@AH-64D Apache Longbow/Addons/fza_ah64_mpd/functions/tsd/fn_tsdScaleHandleControl.sqf

```diff
@@ -0,0 +1,45 @@
+params ["_heli", "_mpdIndex", "_control", "_state", "_persistStage"];
+#include "\fza_ah64_mpd\headers\mfdConstants.h"
+#include "\fza_ah64_mpd\headers\tsd.hpp"
+
+private _showScale = _state get "showScale";
+private _subpage = _state get "subPageVarPage" select 0;
+private _return = false;
```

Comment:
Could you apply a better name to this please? Maybe

`private _interactHandled`